src/misc.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, config):
         self.top_scale = config["plot_top_scale"]
         self.title = config["plot_title"]
         self.legend = config["plot_legend"]
-        self.use_custom_plot_style = False
+        self.use_custom_plot_style = config["plot_use_custom_style"]
```

Comment:
Regarding this my idea initially was to use this variable to identify whether or not to deviate from preferences. This had several merits. You could choose to use a custom style in the preferences and every project, which had had this disabled would automatically update. Querying from config directly would mean we'd set it upon instance creation like the other variables and keep it consistent.
While I like both ways I tend towards the latter.